addons/armaos/functions/fnc_os_mount.sqf

```diff
@@ -0,0 +1,26 @@
+/**
+ * Mounts a filesystem which is connect via a given interface.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: File <[STRING]>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer", "_options"];
+
+if (count _options > 1) exitWith { [ _computer, format [localize "STR_AE3_ArmaOS_Exception_CommandHasTooManyOptions", "mount"] ] call AE3_armaos_fnc_shell_stdout; };
```

Comment:
We should keep in mind to later convert this part to the new getOpts workflow.

Reply:
I will do this, when getOpts is merged